examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -137,12 +139,15 @@ def preprocess(self, requests):
                 context = question_context["context"]
                 inputs = self.tokenizer.encode_plus(question, context, max_length=int(max_length), pad_to_max_length=True, add_special_tokens=True, return_tensors="pt")
             input_ids = inputs["input_ids"].to(self.device)
+            attention_mask = inputs["attention_mask"].to(self.device)
             if input_ids.shape is not None:
```

Comment:
what is this trying to do? quick comment in source code would be helpful

Reply:
@msaroufim this just ensures that we will make a batch out of the requests only when  input_ids exists.